.circleci/config.yml

```diff
@@ -836,98 +836,184 @@ workflows:
-      - sqlalchemy
-      - psycopg
-      - aiobotocore
-      - aiopg
-      - redis
-      - rediscluster
-      - kombu
-      - sqlite3
-      - msgpack
-      - vertica
-      - jinja2
       - build_docs
+      - flake8
+      - aiobotocore:
+          requires:
+            - flake8
+      - aiohttp:
+          requires:
+            - flake8
+      - aiopg:
+          requires:
+            - flake8
+      - asyncio:
+          requires:
+            - flake8
+      - boto:
+          requires:
+            - flake8
+      - bottle:
+          requires:
+            - flake8
+      - cassandra:
+          requires:
+            - flake8
+      - celery:
+          requires:
+            - flake8
+      - ddtracerun:
+          requires:
+            - flake8
+      - django:
+          requires:
+            - flake8
+      - elasticsearch:
+          requires:
+            - flake8
+      - falcon:
+          requires:
+            - flake8
+      - flask:
+          requires:
+            - flake8
+      - futures:
+          requires:
+            - flake8
+      - gevent:
+          requires:
+            - flake8
+      - grpc:
+          requires:
+            - flake8
+      - httplib:
+          requires:
+            - flake8
+      - integration:
+          requires:
+            - flake8
+      - jinja2:
+          requires:
+            - flake8
+      - kombu:
+          requires:
+            - flake8
+      - mongoengine:
+          requires:
+            - flake8
+      - msgpack:
+          requires:
+            - flake8
+      - mysqlconnector:
+          requires:
+            - flake8
+      - mysqldb:
+          requires:
+            - flake8
+      - mysqlpython:
+          requires:
+            - flake8
+      - opentracer:
+          requires:
+            - flake8
+      - psycopg:
+          requires:
+            - flake8
+      - pylibmc:
+          requires:
+            - flake8
+      - pylons:
+          requires:
+            - flake8
+      - pymemcache:
+          requires:
+            - flake8
+      - pymongo:
+          requires:
+            - flake8
+      - pymysql:
+          requires:
+            - flake8
+      - pyramid:
+          requires:
+            - flake8
+      - redis:
+          requires:
+            - flake8
+      - rediscluster:
+          requires:
+            - flake8
+      - requests:
+          requires:
+            - flake8
+      - requestsgevent:
+          requires:
+            - flake8
+      - sqlalchemy:
+          requires:
+            - flake8
+      - sqlite3:
+          requires:
+            - flake8
+      - test_utils:
+          requires:
+            - flake8
+      - tornado:
+          requires:
+            - flake8
+      - tracer:
+          requires:
+            - flake8
+      - vertica:
+          requires:
+            - flake8
       - wait_all_tests:
           requires:
+            - build_docs
             - flake8
```

Comment:
can probably get rid of this requirement now?

Reply:
I wasn't sure about that... but I figured worse case keep it in here in case we re-org again and forget to add it back in 😆 

If you have a stronger opinion about it being removed, happy to, no real reason other than ☝️

Reply:
nope, no pref! :shipit: